### PR TITLE
geant3: new package

### PR DESCRIPTION
**Pull request by aphecetche**

### CODE REVIEW

File: var/spack/repos/builtin/packages/geant3/package.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/geant3/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — APPROVED** [review]
LGTM. Confirmed sha256.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Please fix the copyright year first.

File: var/spack/repos/builtin/packages/geant3/package.py (hunk 1 above)
Comment (aphecetche) [reply var/spack/repos/builtin/packages/geant3/package.py]: done. sorry about that.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**